WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/Onboarding Errors/InPersonPaymentsSelectPluginView.swift

```diff
@@ -0,0 +1,77 @@
+import SwiftUI
+
+struct InPersonPaymentsSelectPlugin: View {
```

Comment:
There are no designs for this screen yet, so I put something together just so that we can unblock ourselves